docs/troubleshoot/troubleshoot.md

```diff
@@ -0,0 +1,238 @@
+    | MongoDB       | `everest` | `kubectl logs -f deploy/percona-server-mongodb-operator -n everest` |
+    | PXC           | `everest` | `kubectl logs -f deploy/percona-xtradb-cluster-operator -n everest` |
+
+    !!! note
+        Update the namespace (`-n <namespace>`) in your commands if your database clusters are deployed in a namespace other than the default one.
+
+3. **Monitoring**
+
+    | Component                | Command                                                                   |
+    | ------------------------ | ------------------------------------------------------------------------- |
+    | VictoriaMetrics Operator | `kubectl logs -f deploy/vm-operator -n everest-monitoring`                |
+    | VM Agent                 | `kubectl logs -f deploy/vmagent-everest-monitoring -n everest-monitoring` |
+
+
+4. **Database and Proxy Pods**
+
+    ```sh
+    kubectl logs -f <pod-name of database or proxy> -c <database-container>
+    ```
+
+5. **Kubernetes Events**
+
+    !!! info "Important"
+        Events (Events are stored only for 60 mins; if there are any events older than 60 minutes, they will be lost).
+
+    ```sh
+    kubectl get events --sort-by=".lastTimestamp"
+    ```
+
+## Troubleshooting key areas
+
+### Installation issues
+
+For troubleshooting Percona Everest installation issues using **everestctl** or the **Helm chart**, the following steps may be helpful:
+{.power-number}
+
+1. **Permissions and privileges**
+
+    Installing database operators and their dependencies may require appropriate privileges. 
+
+     - Percona Everest installation may require `cluster-admin` privileges for:
+        - **Operator Lifecycle Manager (OLM)**
+        - **CustomResourceDefinitions (CRDs)**
+
+    If you encounter failures during installation, ensure your user account has the appropriate permissions. 
+
+    !!! note
+	    Application teams frequently encounter problems when installing operators due to **insufficient privileges**.
+
+    Run the following command to check if the required privileges are granted:
+
+    ```sh
+    kubectl auth can-i <verb> <resource> --namespace <namespace>
+    ```
+
+    To verify if you can create CRDs, run the following command:
+
+    ```
+    kubectl auth can-i create crd
+    ```
+
+    Ensure that your cluster administrator grants the required privileges before you **retry** the installation.
+
+
+2. **Helm Chart validation**
+
+    - Verify the installation status of the Helm chart. A properly functioning chart should be in a **Deployed** status. 
+
+        To verify the values used during the chart installation, run the following command:
+
+        ```sh
+        helm list -n everest-system
+        NAME  NAMESPACE  REVISION  UPDATED                             	       STATUS CHART  APP VERSION
+        everest-core everest-system	1  2025-01-16 16:24:56.577713 +0530 
+        IST	deployed	everest-1.4.0	1.4.0
```

Comment:
Let's try to format this correctly so that the fields are under the correct column